src/openforms/forms/models/utils.py

```diff
@@ -22,11 +25,33 @@ def contribute_to_class(self, cls: ModelBase, name: str) -> None:
 
         # generate the getter function & set it on the class
         def getter(instance: models.Model) -> str:
-            value = getattr(instance, self.model_field)
+            model_field = self.model_field
+            config_field = self.config_field
+            if self.language_code:
+                model_field = f"{self.model_field}_{self.language_code}"
+                config_field = f"{self.config_field}_{self.language_code}"
```

Comment:
```suggestion
            if self.language_code:
                model_field = f"{model_field}_{self.language_code}"
                config_field = f"{config_field}_{self.language_code}"
```

maybe this makes it clearer that `self.model_field` is only a starting point